### PR TITLE
Coverage endpoints

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

**Review by Trjegul84 — COMMENTED** [review]
Loading Caquetá state took about 3 minutes.

**Comment by erikasv** [comment]
There is an error generating the docs

File: services/backend/src/persistence/protected_area.js [hunk 1]
```diff
@@ -313,5 +313,53 @@ module.exports = (db, { globalBinaryProtectedAreas }) => {
...
+     */
+    findCoverageLayer: async (categoryName) => {
+      let bitMask = await globalBinaryProtectedAreas
+        .query()
+        .where({ label: categoryName })
+        .select('binary_protected as mask');
+      bitMask = bitMask[0].mask;
+      return db
+        .raw(
+          `
```
Comment (erikasv) [inline services/backend/src/persistence/protected_area.js:331]: this indentation (in all persistences) seems odd, I think the query should be one level deeper than the opening `` ` ``, or at least the same level

File: services/backend/api_doc/coverages.js [hunk 2]
```diff
@@ -37,3 +37,34 @@
  *    }
  *  ]
  */
+
+/**
+ * @apiDefine CoverageLayerInEAExample
```
Comment (erikasv) [inline services/backend/api_doc/coverages.js:42]: If you are going to use this example in every endpoint, you should name it `CoverageLayerInGeofenceExample`

File: services/backend/src/routes/basin_subzones.js [hunk 3]
```diff
@@ -654,5 +654,34 @@ module.exports = (errorHandler, basinSubzoneService) => {
...
+   * @apiParam (Path params) {Number} subzone_id basin subzone id
+   *
+   * @apiSuccess (geojson) {Object[]} result
+   * @apiSuccess (geojson) {String} result.type The geometry type
+   * @apiSuccess (geojson) {Object[]} result.features features information
+   * (type, properties, geometry)
+   *
+   * @apiExample {curl} Example usage:
+   *  /basinSubzones/3701/coverage/layer
+   * @apiUse CoverageLayerInSubzoneExample
```
Comment (erikasv) [inline services/backend/src/routes/basin_subzones.js:674]: This doesn't exists

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/backend/src/persistence/protected_area.js (hunk 1 above)
Comment (cazapatamar) [reply services/backend/src/persistence/protected_area.js]: Done

File: services/backend/api_doc/coverages.js (hunk 2 above)
Comment (cazapatamar) [reply services/backend/api_doc/coverages.js]: Done

File: services/backend/src/routes/basin_subzones.js (hunk 3 above)
Comment (cazapatamar) [reply services/backend/src/routes/basin_subzones.js]: Done

**Review by erikasv — APPROVED** [review]

**Pull request merged**